packages/coingecko/src/query/schema.graphql

```diff
@@ -1,10 +1,22 @@
 #import { Query } into HTTP from "w3://ens/http.web3api.eth"
-#import { Ping, SimplePrice, SimpleTokenPrice } from "../commons/schema.graphql"
+#import { Ping, CoinsMarkets, SimplePrice, SimpleTokenPrice } from "../commons/schema.graphql"
 
 type Query {
   ping: Ping!
+  
+  coinsMarkets ( 
+    vs_currency:String!
+    id:String
+    category:String
+    order:String
+    per_page: Int
+    page: Int
+    sparkline: Boolean
+    price_change_percentage: [String!]
+    ):[CoinsMarkets!]!
 
-  simplePrice(
+
+    simplePrice(
```

Comment:
format this. feel free to look into auto formatters for graphql if you are interested